SignInCheckIn/MinistryPlatform.Translation/Repositories/Interfaces/IMinistryPlatformRestRepository.cs

```diff
@@ -31,6 +31,8 @@ public interface IMinistryPlatformRestRepository
 
         T Update<T>(T objectToUpdate, List<string> selectColumns);
         T Update<T>(T objectToUpdate, string selectColumns = null);
+        List<T> Update<T>(List<T> objectToUpdate, List<string> selectColumns);
```

Comment:
Your implementation calls the first param object**s**ToUpdate. Should probably be consistent, and plural makes more sense with the list.